howso/client/base.py

```diff
@@ -52,11 +78,19 @@ def get_version(self):
 
     @abstractmethod
     def create_trainee(
-        self, trainee, *,
-        library_type=None,
-        max_wait_time=None,
-        overwrite_trainee=False,
-        resources=None
+        self,
+        name = None,
+        features = None,
+        *,
+        overwrite_trainee = False,
+        persistence = "allow",
+        id = None,
+        library_type = None,
+        max_wait_time = None,
+        metadata = None,
+        project = None,
+        resources = None,
+        client = None
     ):
```

Comment:
Wont need the `client` in this method, since this is already the client.
```suggestion
        resources = None,
    ):
```

Reply:
Addressed by: c28e6de8dded39f04ba3206735fe885f931d8d2e